app/views/admin/editions/show.html.erb

```diff
@@ -0,0 +1,68 @@
+<% content_for :title, @edition.title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds" data-module="track-button-click" data-track-category="button-pressed" data-track-action="<%= @edition.format_name.parameterize(separator: '-') %>-button">
+    <%=
+      status_text = [@edition.state.capitalize]
+      # TODO: remove unpublishing information once we have a separate state for unpublished editions
+      status_text << "unpublished #{time_ago_in_words(@edition.unpublishing.created_at)} ago" if @edition.unpublishing.present?
+
+      render "govuk_publishing_components/components/summary_list", {
+        items: [
+          { field: "Type of document", value: edition_type(@edition) },
+          { field: "Status", value: status_text.join(', ') },
+          { field: "Change type", value: @edition.minor_change? ? 'Minor' : 'Major' },
+          { field: "Organisations", value: joined_list(@edition.organisations.map { |o| o.name }) },
+        ]
+      }
+    %>
+
+    <%=
+      list_items = [
+        link_to("Notes", admin_edition_editorial_remarks_path(@edition), class: "govuk-link"),
+        link_to("History", history_admin_edition_path(@edition), class: "govuk-link"),
+        *(link_to("Fact checking", admin_edition_fact_check_requests_path(@edition), class: "govuk-link") if @edition.can_be_fact_checked?),
```

Comment:
I don't think these 3 links should render anymore now we will be readding tabs in.